config/dependencies.yml

```diff
@@ -18,7 +18,7 @@
 
 auth:
   python:
-    version: '3.0.0'
+    version: '1.4.11'
```

Comment:
You might need to start at 2.0. There was a large breakage there, but if your code doesn't depend on it that isn't such a big deal.